WTWSMS/localisation/WtWSMS_events.csv

```diff
@@ -335,6 +335,11 @@ SENATE.10005.name;Reformation of the Roman Senate;R
 SENATE.10005.desc;After having disappeared from the Roman world, the Roman Senate is now once again back on the political scene of the empire. Senators from the families of old have rejoined the institution and will resume their positions, although the Senate may not for long, if ever, recover from the shattering blow its past disappearance dealt to it.;Apr�s avoir disparu du monde romain, le S�nat romain est maintenant de nouveau de retour sur la sc�ne politique de l'empire. Les s�nateurs des familles d'autrefois ont rejoint l'institution et reprendront leurs fonctions, bien que le S�nat ne puisse pas pour longtemps, voire jamais, se remettre du coup bouleversant que lui a inflig� sa disparition pr�c�dente.;;;Despu�s de haber desaparecido del mundo romano, el Senado romano vuelve a estar en la escena pol�tica del imperio. Los senadores de las familias de anta�o se han reincorporado a la instituci�n y reanudar�n sus cargos, aunque el Senado puede que no se recupere por mucho tiempo del golpe demoledor que le ocasion� su desaparici�n pasada.;;;;;;;x
 SENATE.10005.a;The Senate lives once again!;Le S�nat vit une fois de plus !;;;�El senado vive de nuevo!;;;;;;;x
 SENATE.10005.b;These dusty oldsters aren't going to restore the glory of Rome.;Ces vieillards poussi�reux ne vont pas restaurer la gloire de Rome.;;;Estos viejos polvorientos no van a restaurar la gloria de Roma.;;;;;;;x
+SENATE.10006.name;New consul needed;;;;;;;;;;;x
+SENATE.10006.desc;New consul of the Roman Senate is assigned just like in the days of the Republic: every year or when the current consul dies during his service. Now necessity arose to assign a new consul and you, [Root.GetTitle] of [Root.PrimaryTitle.GetName], seem to have the power to do this.;;;;;;;;;;;x
```

Comment:
Rather than "seem to have the power to do this" I would privilege "you have the power to do this because ..."